### ISSUE TITLE
Mediastinal shift

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10189]
**Preferred term label:**
Mediastinal shift

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0264576

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
COPD, lung cancer, hemothorax

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
def: A deviation of the mediastinum from its normal position in the midline of the thoracic cavity. The mediastinum is a compartment of the thoracic cavity that contains the heart and its blood vessels, the esophagus, trachea, thymus, as well as nerves and lymph nodes.
PMID:35990927|PMID:37680422

Comment: Since the pleural cavity is confined by the rib cage, large pleural effusion may push the structures in the thoracic cavity to the opposite side resulting in a shift of the upper and lower mediastinum. Other etiologies are also observed. See Figures 1 and 2 of PMID:35990927 or Figure 1 of PMID:37680422.

**Comment by MickeySegal** [comment]
Did this get an HPO code?

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000012
- label: Mediastinal shift
- parents: HP:0045026
- pmid: PMID:35990927|PMID:37680422
- def: A deviation of the mediastinum from its normal position in the midline of the thoracic cavity. The mediastinum is a compartment of the thoracic cavity that contains the heart and its blood vessels, the esophagus, trachea, thymus, as well as nerves and lymph nodes.
- comment: Since the pleural cavity is confined by the rib cage, large pleural effusion may push the structures in the thoracic cavity to the opposite side resulting in a shift of the upper and lower mediastinum. Other etiologies are also observed. See Figures 1 and 2 of PMID:35990927 or Figure 1 of PMID:37680422.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10189
- synonyms:
